apps/bidsificator/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# syntax=docker/dockerfile:1
+FROM ubuntu:22.04
+SHELL ["/bin/bash", "-xe", "-o", "pipefail", "-c"]
+
+ARG APP_NAME
+ARG APP_VERSION
+
+WORKDIR /apps/${APP_NAME}
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+    --mount=type=cache,target=/var/lib/apt,sharing=locked \
+    apt-get update -qy && \
+    apt-get install --no-install-recommends -qy \
+    git python3 python3-pip \
+    # lib necessary to have a working pyqt application \
+    libxcb-icccm4 libxcb-randr0 libxcb-render-util0 \
+    libxcb-shape0 libxcursor1 libxkbcommon-x11-0 \
+    libxcb-keysyms1 libglib2.0-0 libdbus-1-3 libxcb-icccm4 \
+    libxcb-randr0 libxcb-render-util0 libxcb-shape0 fontconfig \
+    libfontconfig1 libxcb-image0 libxcb-util1 && \
+    # install using pip and git \
+    pip install --no-cache git+https://github.com/HIP-infrastructure/Bidsificator/@V${APP_VERSION}#egg=Bidsificator && \
```

Comment:
https://github.com/HIP-infrastructure/Bidsificator/blob/V1.0.0/pyproject.toml#L3 :wink:

Reply:
oh i forgot this existed ahah

Reply:
okidoki this was updated and i switched to lowercase v for version numbering